packages/sanity/src/core/releases/components/Table/TableHeader.tsx

```diff
@@ -1,4 +1,5 @@
 import {ArrowDownIcon, ArrowUpIcon, SearchIcon} from '@sanity/icons'
+// eslint-disable-next-line no-restricted-imports -- Table header using fine-grained styling
```

Comment:
I think we could use the internal <Button> here. Unless we really want to keep a different radius and spacing prop.
But the intention of the opinionated button is to normalize all the possible usages of buttons in the ui.
This is how it would look using the internal `<Button>`
<img width="597" alt="Screenshot 2024-08-19 at 12 09 35" src="https://github.com/user-attachments/assets/fe1ceb43-440b-4f9b-ace7-f7b5e6214a99">

Code changes:

```

const SortHeaderButton = ({
  header,
  text,
}: Omit<ButtonProps, 'text'> &
  HeaderProps & {
    text: string
  }) => {
  const {sort, setSortColumn} = useTableContext()
  const sortIcon = sort?.direction === 'asc' ? ArrowUpIcon : ArrowDownIcon

  return (
    <Button
      iconRight={header.sorting && sort?.column === header.id ? sortIcon : undefined}
      onClick={() => setSortColumn(String(header.id))}
      mode="bleed"
      size="default"
      text={text}
    />
  )
}

```

Reply:
- [ ] Use ui-component Button for Table headers